app/src/main/java/org/piramalswasthya/cho/ui/login_activity/login_settings/LoginSettingsFragment.kt

```
@@ -238,13 +238,13 @@ class LoginSettingsFragment : Fragment() {
                             viewModel.userInfo?.userId
                         )
                     )
-                    Toast.makeText(activity, "Data is Saved!", Toast.LENGTH_SHORT).show()
+                    Toast.makeText(activity, getString(R.string.data_saved), Toast.LENGTH_SHORT).show()
                 }catch(e:Exception){
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)